contracts/ethereum/contracts/LootboxStaking.sol

```diff
@@ -33,19 +33,19 @@ import "./openzeppelin/SafeERC20.sol";
  * - SushiSwap
  *
  * Divided 50/50
-*/
+ */
 
 uint256 constant MAX_UINT256 = type(uint256).max;
 
-uint256 constant MIN_LOCKUP_TIME = 31 days;
-
-uint256 constant MAX_LOCKUP_TIME = 365 days;
-
 uint256 constant MAX_SLIPPAGE = 100;
 
 contract LootboxStaking is ILootboxStaking, IOperatorOwned, IEmergencyMode {
     using SafeERC20 for IERC20;
 
+    uint256 private minLockupTime_;
```

Comment:
These changes have to be at the bottom of the states due to the upgrade compatibility, and you can't change the constants instead?